### DISCUSSION TITLE
My Release Plan

### DISCUSSION THREAD

**Discussion by boozook** [pd-rs/crankstart#55] [Announcements]
Currently in draft state but already in work :))

[This](https://github.com/users/boozook/projects/1)

**Comment by boozook** [comment]
Tomorrow. Because I'm so exhausted and I need help with it by community, by you.
Also I've discussed with Panic about "playdate" name usage in crates names and still get no any concrete result after more then month. But other persons freely publishing crates with "playdate" in crate name. I feeling myself really stupid dumb. Many things could be better.
Well, tomorrow anyway.

**Reply by jcornaz** [reply]
It's fantastic that you contacted them about the legal usage of the word "playdate"! Don't beat yourself up for doing your best.

However, "playdate" is an [English word](https://www.dictionary.com/browse/playdate). I am not a lawyer, but I doubt they can decide anything about how that word can and cannot be used.

Especially if you honestly do nothing but promote their business by developing free tools.

The only thing that could *theoretically* happen is: If they eventually decide to publish their own "official" rust crate and would ask you to transfer ownership of the crate name. But, that's unlikely and would be great news anyway IMO :-)

**Reply by boozook** [reply]
Unfortunately not today. Tomorrow or in couple days. I've just started to watch Chinese "Three-Body" that I waited long time. I really love the books The Three-Body Problem. And so today world lost me for a day. 🤷🏻‍♂️🤦🏻‍♂️

**Comment by boozook** [comment]
Here we go! 🎉
[THIS](https://github.com/boozook/playdate)

**Reply by jcornaz** [reply]
Awesome! I am looking forward to finding the time to try that out :-)

Thanks for your efforts 👍

**Reply by boozook** [reply]
Also probably crankstart can be built with cargo-playdate without any changes. I haven't tried yet but it's in plan.
